README.md

```diff
@@ -1,4 +1,31 @@
-# Simple-Calculator-Web-app
+# Simple-Calculator-Web-app -- An Amazing Project
+
+
+The most simple mathematical operations: addition, subtraction, multiplication, division. This is a simplified version of the online calculator.
+
+
+
+# Entering commands from the keyboard
+For the work of the simple calculator You can use both ordinary numeric buttons above the keyboard and separate numeric buttons on the right.
+
+You can use the [Enter] key to enter the equal character.
+
+To delete the last character, press the [Backspace] (arrow) key.
+
+To enter a plus sign, use the [ + ] key at the top or on the side numeric keypad.
+
+To enter a minus sign, use the [ - ] key at the top or on the side numeric keypad.
+
+Use the [ * ] key on the side numeric keypad to enter the multiplication sign.
+
+Use the [ / ] key on the side numeric keypad to enter the division sign.
+
+You can use the [Esc] and [Del] keys on top of the keyboard, or the [End] key on the side numeric keypad to reset the calculator.
+
+
+
+# The full version you can get and use here: Calculator
```

Comment:
The above description is correct and looks good,
what about this line though?

`# The full version you can get and use here: Calculator` ?

Was there a link you wanted to add for `Calculator` ? if not, maybe remove the line